.changeset/shaggy-snakes-sip.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
what does this file do?

Reply:
This is for the change-log when we release a new version of the packages - see the changeset package

Reply:
cool